images/tidb-backup-manager/entrypoint.sh

```diff
@@ -51,33 +51,40 @@ else
 fi
 
 BACKUP_BIN=/tidb-backup-manager
+if [[ -n "${AWS_DEFAULT_REGION}"]]; then
```

Comment:
this env `AWS_DEFAULT_REGION` is always injected into pods on eks?

Reply:
this AWS_DEFAULT_REGION only use in useKMS is on